src/audio-worklet/design-pattern/lib/wasm-audio-helper.js

```diff
@@ -249,14 +247,12 @@ class RingBuffer {
 
     // Transfer data from the internal buffer to the |arraySequence| storage.
     for (let i = 0; i < destinationLength; ++i) {
-      const readIndex = (this._readIndex + i) % this._length;
+      this._readIndex = (this._readIndex + i) % this._length;
```

Comment:
Same as above, we should just be incrementing by `1` instead of `i` now that we're storing back to the class variable.